src/main/java/net/mtrop/doom/tools/WTexScanMain.java

```diff
@@ -236,7 +246,8 @@ private void inspectWAD(Wad wad) throws IOException
 		{
 			String[] mapHeaders = MapUtils.getAllMapHeaders(wad);
 			for (String mapName : mapHeaders)
-				inspectMap(wad, mapName);
+				if(options.mapsToScan.isEmpty() || options.mapsToScan.contains(mapName))
```

Comment:
Teensy, tiny nitpick: `if` should have a space after it for code style conformity.